tsconfig.json

```diff
@@ -1,13 +1,20 @@
 {
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Node 16",
+  "extends": "@tsconfig/node16/tsconfig.json",
   "compilerOptions": {
-    "target": "ES2019",
+    "lib" : ["es2021"],
+    "target": "es2021",
     "module": "commonjs",
     "esModuleInterop": true,
     "declaration": true,
     "declarationMap": true,
     "sourceMap": true,
     "removeComments": false,
     "strict": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
+    "moduleResolution": "node",
```

Comment:
Are these changes necessary for node16?  Do you have related document link?

Reply:
Should it be `"extends": "@tsconfig/node16-strictest/tsconfig.json"`?

Reply:
and https://www.npmjs.com/package/@tsconfig/node16-strictest

Reply:
I think only `"target": "es2021",` is necessary.  The other options change the current typescript setting. @Jkovarik any suggestion?